lib/rucio/common/exception.py

```diff
@@ -811,3 +820,13 @@ def __init__(self, *args, **kwargs):
         super(MissingModuleException, self).__init__(args, kwargs)
         self._message = "The module is not installed."
         self.error_code = 75
+
+
+class RucioServerException(RucioException):
```

Comment:
I think this should be renamed to ConnectionFailureException. Or FailedConnectionException or something like this.

Reply:
I thought about it but failure/Failed sounds to me like an temporary glitch and that it can be retried later on while here it can be a misconfiguration for example. I'm looking for a better name...